src/test/java/ay2122s1_cs2103t_w16_2/btbb/exception/CommandExceptionTest.java

```diff
@@ -0,0 +1,20 @@
+package ay2122s1_cs2103t_w16_2.btbb.exception;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class CommandExceptionTest {
+    private static final String COMMAND_EXCEPTION_MESSAGE = "This is a CommandException.";
+
+    @Test
+    public void constructor_throwCommandException_commandExceptionThrown() {
```

Comment:
Minor thing but is it better to just name exception tests as testXXException?
Because it doesn't seem like throwXException is a test input, so it seems a bit different from the format of whatIsBeingTested_descriptionOfTestInputs_expectedOutcome